core/src/main/resources/reference.conf

```diff
@@ -368,6 +368,11 @@ docker {
     max-time-between-retries = 1 minute
     max-retries = 3
 
+    // If the docker digest is provided, disable the lookup from the registry. This request can fail on environments
+    // where a web proxy is required - which would disable call-caching.
+    // Note, this request also gets the image size, which is required on the Google backend to adjust boot disk size.
+    perform-registry-lookup-if-digest-is-provided = true
```

Comment:
Could you add a note to say something like "This option is provided as a community contribution. Switching the value to 'false' is NOT officially supported by the Cromwell team."

And if you're feeling brave, you could also add: "Please contact xyz with any questions about this functionality"

Reply:
Thanks @cjllanwarne, this comment now reads:

```
// COMMUNITY FEATURE:
// If the docker digest is provided, disable the lookup from the registry. This request can fail on environments
// where a web proxy is required - which would disable call-caching.
// Note, this request also gets the image size, which is required on the Google backend to adjust boot disk size.
// This option is provided as a community contribution. Switching the value to 'false' is NOT officially supported by the Cromwell team.
// Please contact illusional (Michael Franklin) with any questions about this functionality
perform-registry-lookup-if-digest-is-provided = true
```